apps/nextjs/src/middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { authMiddleware } from "@clerk/nextjs";
+
+// This example protects all routes including api/trpc routes
+// Please edit this to allow other routes to be public as needed.
+// See https://clerk.com/docs/references/nextjs/auth-middleware for more information about configuring your Middleware
+export default authMiddleware();
+
+// Do not protect '/' for now since that's where we land
+export const config = {
+  matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/(api|trpc)(.*)"],
+};
```

Comment:
I've been getting a warning while building that a package isn't edge friendly -- do you know if @clerk/nextjs works on edge / do the docs say anything?

Reply:
okay found a fix -- change the version of `@clerk/nextjs` to "4.25.5" in the `package.json`. Not sure why this works but it does...

Reply:
I don't believe so unless you're using `withEdgeMiddlewareAuth`? I can't find anything else that references edge in the docs

Reply:
I think a fix is to bump up `@clerk/next` to "^4.26"

Reply:
Updated